code/modules/mob/living/carbon/alien/special/alien_embryo.dm

```diff
@@ -101,8 +101,14 @@
 			new_xeno << sound('sound/voice/hiss5.ogg',0,0,0,100)//To get the player's attention
 			to_chat(new_xeno, "<span class='motd'>For more information, check the wiki page: ([GLOB.configuration.url.wiki_url]/index.php/Xenomorph)</span>")
 
-			if(gib_on_success)
-				owner.gib()
+			if(kill_on_success)
+				var/obj/item/organ/internal/heart/E = owner.get_int_organ(/obj/item/organ/internal/heart)
+				if(istype(E))
+					E.receive_damage(50)
+				owner.death()
+				owner.overlays -= overlay
+				var/overlay_dead = image('icons/mob/alien.dmi', loc = owner, icon_state = "bursted_lie")
+				owner.overlays += overlay_dead
```

Comment:
Do these overlays get cleared if the person is healed/revived? You don't want people walking around with gaping holes (or at least that's what I assume the icon is).

Reply:
I believe so. When the sprite is updated from a damage one to an undamaged one from healing the overlays seem to disappear, but I also found them to sometimes be buggy. I might've messed up code-wise here and this could use a fix. I'll check it out more thoroughly and update the PR with any changes if they are necessary.

Reply:
> Do these overlays get cleared if the person is healed/revived? You don't want people walking around with gaping holes (or at least that's what I assume the icon is).

Oop! Just did a bunch of testing, and no it does not go away aside from a clone or an aheal. I'm not sure how the best way to fix it, or when the best way to fix it is (when defibbed or something? idk). Will need some help on this one.